packages/keystone/lib/Keystone/index.js

```diff
@@ -268,6 +268,13 @@ module.exports = class Keystone {
       throw new Error(`Invalid list name "${key}". List names cannot start with an underscore.`);
     }
 
+    // Apollo Server automatically adds an 'Upload' scalar type to the GQL schema. Since list output
+    // types are named after their keys, having a list name 'Upload' will clash and cause a confusing
+    // error on start.
+    if (key === 'Upload' || key === 'upload') {
+      throw new Error(`"Upload" cannot be used as a list name.`);
```

Comment:
```suggestion
      throw new Error(`Invalid list name "Upload": Built-in GraphQL types cannot be used as a list name.`);
```

Reply:
For consistency with the error message on line 268 and also to explain why `Upload` cant be used. This of course raises the question of whether we should check for other built in names like `Query`, `Mutation`, `String`, etc 🤔

Reply:
Done.